src/Protocol/OStatus.php

```diff
@@ -1023,6 +1024,7 @@ private static function processRepeatedItem($xpath, $entry, &$item, $importer)
 		$orig_created = $xpath->query('atom:published/text()', $activityobjects)->item(0)->nodeValue;
 		$orig_edited = $xpath->query('atom:updated/text()', $activityobjects)->item(0)->nodeValue;
 
+		/// @fixme $contact is unavailable here
 		$orig_contact = $contact;
```

Comment:
Just remove this line and replace ```orig_contact``` in the following line with ```contact```. I guess this is some leftover from the time that this functionality wasn't in a dedicated function.

Reply:
The problem is that there's no `$contact` at all.

Reply:
This is no problem. It is called by reference.